src/client/Microsoft.Identity.Client/MsalError.cs

```diff
@@ -793,6 +793,11 @@ public static class MsalError
         /// </summary>
         public const string AuthorityTypeMismatch = "authority_type_mismatch";
 
+        /// <summary>
+        /// <para>What happens?</para>You specified a tenant twice - once in WithAuthority() and once using WithTenant()
+        /// <para>Mitigation</para>Specify the tenant once.
+        /// </summary>
+        public const string AuthorityTenantSpecifiedTwice = "authority_type_mismatch";
```

Comment:
The value of the constant AuthorityTenantSpecifiedTwice is same as that of AuthorityTypeMismatch.

Reply:
Good catch